### PR TITLE
fix / avellaneda status msg

### PR DESCRIPTION
**Pull request by keithbaum**
Changed condition to check for "volatility is not NaN" since when volatility=0 although being valid number "bool(volatility) =False" and status message was not showing. 

**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

### CODE REVIEW

**Comment by RC-13** [comment]
Tested and confirmed that PR fix the reported issue. 
Liquid:
image
Kraken:
image
Observe the bots for 1 hour and the strategy parameters doesn't go away

**Review by RC-13 — DISMISSED** [review]

File: hummingbot/strategy/avellaneda_market_making/avellaneda_market_making.pyx [hunk 1]
```diff
@@ -355,7 +355,7 @@ cdef class AvellanedaMarketMakingStrategy(StrategyBase):
             lines.extend(["", "  No active maker orders."])
 
         volatility_pct = self._avg_vol.current_value / float(self.get_price()) * 100.0
-        if all((self._gamma, self._kappa, volatility_pct)):
+        if all((self._gamma, self._kappa, volatility_pct is not NaN)):
```
Comment (CrimsonJacket) [inline hummingbot/strategy/avellaneda_market_making/avellaneda_market_making.pyx:358]: I don't think this is correct. I do know that checking if a variable is `NaN`, we can't just use `a is not NaN` or `a != NaN`. We should use `math.isnan()` or its equivalent.

image

**Review by CrimsonJacket — CHANGES_REQUESTED** [review]
I don't think this is correct. I do know that checking if a variable is `NaN`, we can't just use `a is not NaN` or `a != NaN`. We should use `math.isnan()` or its equivalent.

image

File: hummingbot/strategy/avellaneda_market_making/avellaneda_market_making.pyx (hunk 1 above)
Comment (keithbaum) [reply hummingbot/strategy/avellaneda_market_making/avellaneda_market_making.pyx]: Change applied

**Review by PtrckM — APPROVED** [review]
Re-test no issue found (run time 1hr), reapproving
image

**Review by CrimsonJacket — APPROVED** [review]
LGTM

**Pull request merged**